agrirouter-api-dotnet-standard-impl/Service/Messaging/ListEndpointsUnfilteredService.cs

```diff
@@ -13,9 +13,7 @@ public class ListEndpointsUnfilteredService : ListEndpointsBaseService
         /// Constructor.
         /// </summary>
         /// <param name="messagingService"></param>
-        /// <param name="encodeMessageService"></param>
-        public ListEndpointsUnfilteredService(MessagingService messagingService,
-            EncodeMessageService encodeMessageService) : base(messagingService, encodeMessageService)
+        public ListEndpointsUnfilteredService(HttpMessagingService messagingService) : base(messagingService)
```

Comment:
Why are you using HttpMessagingService here; in all other classes it's the general IMessagingService